src/DiffViewerProvider.ts

```diff
@@ -118,4 +134,16 @@ export class DiffViewerProvider implements vscode.CustomTextEditorProvider {
       destination: "app",
     });
   }
+
+  private handleOpenFileMessage(message: OpenFileMessage, diffDocument: vscode.TextDocument) {
+    const filePath = path.join(path.dirname(diffDocument.uri.fsPath), message.path);
+    const uri = vscode.Uri.file(filePath);
+
+    const options: vscode.TextDocumentShowOptions = {};
+    if (message.line != null) {
+      options.selection = new vscode.Range(message.line - 1, 0, message.line - 1, 0);
+    }
+
+    vscode.commands.executeCommand("vscode.open", uri, options);
```

Comment:
If we can check the file doesn't exist, it will make sense to enable or disable the clicks also for old files (in case we actually have a diff between two existing files). I'm looking into it; I'm on vacation though so have maybe an hour a day for this.